driver.go

```diff
@@ -156,8 +159,32 @@ func (l *LinstorDriver) newParams(name string, options map[string]string) (*Lins
 		return nil, err
 	}
 
-	err = decoder.Decode(options)
-	return params, err
+	if err := decoder.Decode(options); err != nil {
+		return nil, err
+	}
+
+	// convert string Size to SizeKiB
+	if params.Size == "" {
+		params.Size = "100MB"
+	}
+	u := unit.MustNewUnit(unit.DefaultUnits)
+	strSize := params.Size
+	v, err := u.ValueFromString(strSize)
+	if err != nil {
+		return nil, fmt.Errorf("Could not convert '%s' to bytes: %v", strSize, err)
+	}
+	bytes := v.Value
+	lower := 4 * unit.M
+	if bytes < lower {
+		bytes = lower
+	}
+	params.SizeKiB = uint64(bytes / unit.K)
+
+	if params.FS == "" {
+		params.FS = "ext4"
+	}
+
```

Comment:
Moving my replica options check `params.Replicas == 0 { ... }` might be cleaner see lines 222-224

Reply:
mhm. will add that on top I guess. I will send an updated PR soon